src/components/common/ClientIntakeForm/ClientIntakeForm.js

```diff
@@ -18,15 +19,21 @@ const sectionContainer = {
   margin: 'auto',
 };
 
-const ClientIntakeForm = () => {
+const ClientIntakeForm = onChange => {
   const [form] = Form.useForm();
 
-  //onFinish is AntDs version on onSubmit;
   const onFinish = values => {
     console.log('Form Values: ', values);
     form.resetFields();
   };
 
+  const [fields, setFields] = useState([
+    {
+      name: ['name'],
+      value: 'Chris',
+    },
+  ]);
+
   return (
     <div>
       {/* <ClientSearch /> */}
```

Comment:
Are these empty {} for things we will add in the future?

Reply:
Yes, I believe client search will be added by another colleague.